src/pages/EditRequestPage.js

```diff
@@ -35,27 +36,74 @@ const defaultProps = {
     report: {},
 };
 
-function EditRequestPage(props) {
-    const parentReportAction = ReportActionsUtils.getParentReportAction(props.report);
-    const moneyRequestReportAction = ReportUtils.getMoneyRequestAction(parentReportAction);
-    const transactionDescription = moneyRequestReportAction.comment;
-    const field = lodashGet(props, ['route', 'params', 'field'], '');
+function EditRequestPage({report, route}) {
+    const transactionID = lodashGet(ReportActionsUtils.getParentReportAction(report), 'originalMessage.IOUTransactionID', '');
+    const transaction = TransactionUtils.getTransaction(transactionID);
+    const transactionDescription = TransactionUtils.getDescription(transaction);
+    const transactionAmount = TransactionUtils.getAmount(transaction);
```

Comment:
This amount is negative for expense reports.

![Screenshot 2023-08-10 at 3 09 39 PM](https://github.com/Expensify/App/assets/22219519/bbddbe2b-3b78-42d9-88fc-a5bf3874b309)

Reply:
fixed